.flake8

```diff
@@ -7,5 +7,5 @@ exclude =
     .git,
     __pycache__,
 # E203: Whitespace before ':'. Addresses a discrepancy with Black slice formatting.
-ignore = E203, W503
+ignore = E203, W503, W605
```

Comment:
nit: can we add quick comments for `W503` and `W605` above (similar to the `E203` explanation)?